parquet-generator/src/main/java/org/apache/parquet/filter2/IncrementallyUpdatedFilterPredicateGenerator.java

```diff
@@ -1,14 +1,14 @@
-/* 
```

Comment:
We have to validate In and NotIn for the record level filtering as well. All the tests here are validating higher levels only. (Record level filtering is when you actually read the data and check the values one by one to return only those values that are fulfill the filter. This is done by the class `IncrementallyUpdatedFilterPredicateBuilder` generated by this one.) 

In would suggest checking and extending the test classes `TestRecordLevelFilters`, `TestColumnIndexFiltering` and `TestBloomFiltering`.

Reply:
I will think about how to achieve this. Thanks for the suggestion.

Reply:
@gszadovszky I added a test in `TestRecordLevelFilters.java` to test the new methods in the generated class `IncrementallyUpdatedFilterPredicateBuilder`. I have added tests in `TestColumnIndexFiltering` and `TestBloomFiltering` in my original changes. Do I need more tests in these two files?

Reply:
`TestColumnIndexFiltering` is another test class different from `TestColumnIndexFilter`. The first one is a higher level test. It would be nice if you could add some tests there as well.

Reply:
Added. Thanks!